test/1_stdlib/Availability.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %target-run-simple-swift
+// REQUIRES: executable_test
+
+import StdlibUnittest
+import SwiftShims
+
+let AvailabilityTests = TestSuite("Availability")
+AvailabilityTests.test("==, <, >=") {
```

Comment:
We have tests for this API in `test/1_stdlib/Runtime.swift` (`AvailabilityVersionsTestSuite`).

Could you check if any of these cases are not covered by those tests, and merge them if so?